tests/t283_agent_time.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+import subprocess as sp
+from time import sleep
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    """Test changing the time threshold in the target from the agent.
+
+    The target itself has an original threshold of 2ms. We decrease the time to
+    1ms and then increase it to 3ms."""
```

Comment:
Please also mention that the target program (t-agent) would sleep for 1ms on each function (a, b, c) when --delay option is given.